docs/administration.unmanaged-tenants.md

```diff
@@ -0,0 +1,30 @@
+---
+id: administration.unmanaged-tenants
+title: Unmanaged Tenants
+---
+
+## What is an unmanaged tenant?
+
+Workload that is running indefinitely without anyone's awareness can be an easy way to burn through cloud budget. This is otherwise
+known as 'Shadow IT'. To make it easier to spot this kind of workload, meshStack offers a table view of all tenants that are unmanaged.
+
+The definition of an unmanaged tenant is a cloud tenant that is not related to any meshCustmer & meshProject. In other words, it does not
+have any organizational metadata applied to it, and it is "unknown" who owns the tenant from a meshStack perspective.
+
+> If you recently started using meshStack, you will most likely have a lot of unmanaged tenants. This is fine
+> as you are still working on starting to manage these tenants via meshStack.
+
+## Viewing unmanaged tenants
+
+Viewing unmanaged tenants can be easily done within the administration area. In the sidebar on the left, navigate to 'Platforms' -> 'Unmanaged Tenants'.
+
+A table will open up with all unmanaged tenants that are known to meshStack. This list is refreshed on a daily basis.
+
+It is also recorded when the unmanaged tenant was last observed. If the last observed date is from longer than a few days ago, it has dissappeared. This
+probably means that it was deleted and it no longer exists in the cloud platform.
+
+Tenants that are imported into the meshStack organizational model (meaning into a meshProject) are automatically removed from the list.
```

Comment:
We say "to adopt a cloud tenant" when speaking about this.
```suggestion
Tenants that are adopted into meshStack are removed from umanaged tenants list and shown in the tenants list instead.
```

Reply:
A link to "How to adopt an unmanaged tenant into meshStack" would be great (but I think there is no how-to content for this at the moment).

Reply:
I briefly mentioned the API for now. A how-to would be great in future ;-)